lib/grp.gd

```diff
@@ -1972,7 +1972,6 @@ DeclareAttribute( "NrConjugacyClasses", IsGroup );
 ##  \{ g \in <A>G</A> \mid g^{{<A>p</A>^{<A>n</A>}}} = 1 \}</M>.
 ##  The default value for <A>n</A> is <C>1</C>.
 ##  <P/>
-##  <E>@At the moment methods exist only for abelian <A>G</A> and <A>n</A>=1.@</E>
```

Comment:
OK, I tried this and it seemed to work:
```
gap> grps:=AllSmallGroups(32);;
gap> Collected(List(grps,g->Size(Omega(g,2))));
[ [ 2, 2 ], [ 4, 14 ], [ 8, 10 ], [ 16, 13 ], [ 32, 12 ] ]
gap> Collected(List(grps,g->Size(Omega(g,2,2))));
[ [ 4, 1 ], [ 8, 3 ], [ 16, 9 ], [ 32, 38 ] ]
gap> Collected(List(grps,g->Size(Omega(g,2,3))));
[ [ 8, 1 ], [ 16, 2 ], [ 32, 48 ] ]
```
So it seems you are right. But perhaps add a minimal test to verify this? Anyway, that's optional.